clients/js/proton_compiled.js

```diff
@@ -0,0 +1,1340 @@
+//Generated by the protobufjs.  DO NOT EDIT!
```

Comment:
Can we move it under a folder called protos, instead of having a top level file

Reply:
okay